kiwi/system/root_import/__init__.py

```diff
@@ -47,7 +48,8 @@ class RootImport(metaclass=ABCMeta):
     def __init__(self) -> None:
         return None  # pragma: no cover
 
-    def new(root_dir: str, image_uri: str, image_type: str):
+    @staticmethod
+    def new(root_dir: str, image_uri: Uri, image_type: str):
```

Comment:
Good catch! This was wrong and also with an inconsistent type :+1:

Reply:
yes you'll find all those glitches with mypy :)